test/test-metrics.c

```diff
@@ -130,6 +141,98 @@ TEST_IMPL(metrics_idle_time_zero) {
   ASSERT_GT(cntr, 0);
   ASSERT_EQ(0, uv_metrics_idle_time(uv_default_loop()));
 
+  ASSERT_EQ(0, uv_metrics_info(uv_default_loop(), &metrics));
+  ASSERT_EQ(cntr, metrics.loop_count);
```

Comment:
Use the `ASSERT_UINT64` here and in the rest of the file when it applies (for the metrics)
```suggestion
  ASSERT_UINT64_EQ(cntr, metrics.loop_count);
```